web/business-registry-dashboard/.env.sample

```diff
@@ -5,6 +5,7 @@ NUXT_REGISTRIES_SEARCH_API_KEY=""
 NUXT_X_API_KEY=""
 
 # auth api
+NUXT_AUTH_API_URL="https://auth-api-dev-142173140222.northamerica-northeast1.run.app"
```

Comment:
Looks like this is needed after all...

<img width="1200" height="287" alt="image" src="https://github.com/user-attachments/assets/63848e74-a58f-4b14-af15-835ccb1a6760" />

Reply:
This fix is deployed and working properly.

Summary: this key is indeed required. Not by the BRD code, but apparently by the [Nuxt Core Layer (SBC Connect Common)](https://www.npmjs.com/package/@daxiom/nuxt-core-layer-test) that is imported via package.json.